### PR TITLE
loading jquery only if it does not exist

### PR DESCRIPTION
**Pull request by debopamsengupta**
this should solve #8 

cc @nickbalestra @mattiaerre

### CODE REVIEW

**Comment by mattiaerre** [comment]
hi @debopamsengupta thanks for this; have you been able to test it locally? if so how did you do that?
thanks,

File: src/oc-client.js [hunk 1]
```diff
@@ -328,21 +328,22 @@ var oc = oc || {};
...
 
+      if (wasJqueryThereAlready) {
+        var instanceOfJquery = $window.jQuery || $window.$;
+        requirePolyfills(instanceOfJquery, function() {
+          oc.$ = instanceOfJquery;
           done();
         });
-      });
+      } else {
+        oc.require('jQuery', JQUERY_URL, function(jQuery) {
```
Comment (nickbalestra) [inline src/oc-client.js]: I'm not sure we need an `if/else` as `oc.require` already does some of [this logic](https://github.com/opencomponents/oc-client-browser/blob/master/src/oc-client.js#L208-L215), meaning that it won't fetch `jQuery` if already available.

So perhaps is enough to simply extend the `wasJqueryThereAlready` check as you suggested while leaving the rest of the logic un touched? What you think @debopamsengupta ?

Comment (debopamsengupta) [inline src/oc-client.js]: ah, missed that part in the `oc.require` code, made the changes :)

**Comment by debopamsengupta** [comment]
@mattiaerre I was testing the script locally on a test page....the current tests are probably not geared towards testing the loading of jQuery based on existing page settings

**Comment by nickbalestra** [comment]
Awesome, thanks 💯  @debopamsengupta !

**Comment by debopamsengupta** [comment]
@nickbalestra does the existing `oc-client` package need to be bumped as well ? to use new `oc-client-browser` ?

**Comment by nickbalestra** [comment]
@debopamsengupta yes i see that we didn't enabled greenkeeper on those repos, I've updated let see if it works, otherwise I'll do it manually. Thanks for the heads up

**Pull request merged**